modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -73,7 +73,7 @@ It is recommended to move your vSphere cluster to the VMC `Compute-ResourcePool`
 
 [NOTE]
 ====
-You cannot use the VMware NSX Container Plugin for Kubernetes (NCP), and NSX is not used as the OpenShift SDN. The version of NSX currently available with VMC is incompatible with the version of NCP certified with {product-title}.
+You cannot use the VMware NSX Container Plug-in for Kubernetes (NCP), and NSX is not used as the OpenShift SDN. The version of NSX currently available with VMC is incompatible with the version of NCP certified with {product-title}.
```

Comment:
This aligns with [VMware's use](https://docs.vmware.com/en/VMware-NSX-T-Data-Center/3.1/ncp-kubernetes/GUID-52A92986-0FDF-43A5-A7BB-C037889F7559.html).